addons/mass_mailing_sms/wizard/mailing_sms_test.py

```diff
@@ -32,7 +39,13 @@ def action_send_sms(self):
             # Returns a proper error if there is a syntax error with qweb
             body = self.env['mail.render.mixin']._render_template(body, self.mailing_id.mailing_model_real, record.ids)[record.id]
 
-        new_sms_messages_sudo = self.env['sms.sms'].sudo().create([{'body': body, 'number': number} for number in sanitized_numbers])
+        # Include the unsubscribe link if sms_allow_unsubscribe is enabled
+        if self.mailing_id.sms_allow_unsubscribe:
+            trace_code = self.env['mailing.trace']._get_random_code()
+            unsubscribe_url = self._get_unsubscribe_url(self.mailing_id.id, trace_code)
+            body = '%s\n%s' % (body, _('STOP SMS: %s', unsubscribe_url))
+
+        new_sms_messages_sudo = self.env['sms.sms'].sudo().create([{'body': body, 'number': number} for number in (invalid_numbers if invalid_numbers else sanitized_numbers)])
```

Comment:
This is not the right way to do. 
If we have 2 numbers 1 is invalid and 1 is sanitized in that case It will only log for invalid ones and not for sanitized ones.

Reply:
Oh okay you're right seems like sanitized number is returning False value we don't need that for new_sms_messages_sudo.